### DISCUSSION TITLE
Question regarding the security fix in 2.6.1

### DISCUSSION THREAD

**Discussion by mdosch** [ProtonMail/gopenpgp#236] [Q&A] [answered]
Dear gopenpgp devs,

was the security issue fixed in 2.6.1 introduced in 2.6.0 or is it also present in 2.5.x?

**Comment by twiss** [comment, marked as answer]
Hey :wave: It was there for a while, including in 2.5.x. However, note that the impact of the issue was "merely" that parsing untrusted input could cause a panic (which is of course still bad) - this was marked as a security issue because of the potential for denial of service attacks, there were no other security implications beyond that.

**Comment by mdosch** [comment]
Thank you very much for the clarification. I think I'm going to check whether it makes sense to fix these issues in Debian Bookworm.
Are the related commits https://github.com/ProtonMail/go-mime/commit/c287644401b0141618c993df408c045dc46df8ad and https://github.com/ProtonMail/go-crypto/commit/623e674abdb153c9a67c54022b17bd3b94d31686 ?

**Reply by twiss** [reply]
No problem! And yes, indeed :+1:
